samples/StandaloneMvcClientSample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Diagnostics;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using StandaloneMvcClientSample.Models;
+
+namespace StandaloneMvcClientSample.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            if (!HttpContext.User.Identities.First().IsAuthenticated)
```

Comment:
Should we maybe use the `Authorize` attribute here instead? I know that the focus of this PR is simplicity, but this Might be considered bad practice.

Reply:
Good point, I agree, will rewrite it